Misc/NEWS.d/next/Library/2024-10-16-04-50-53.gh-issue-125542.vZJ-Ns.rst

```diff
@@ -0,0 +1 @@
+Deprecate passing prefix_chars_ parameter to :meth:`ArgumentParser.add_argument_group`.
```

Comment:
I do not think that it works cross files.

Functions are defined with parameters. Arguments are passed to functions. https://docs.python.org/3/faq/programming.html#faq-argument-vs-parameter

```suggestion
Deprecate passing keyword-only :ref:`prefix_chars` argument to :meth:`argparse.ArgumentParser.add_argument_group`.
```

Reply:
Yeah, I'm always battling Sphinx. Thanks!